app/views/partials/theme/simple/marketplace_setup/index.liquid

```diff
@@ -31,3 +31,36 @@
       <button class="mt-3 button">{{ 'app.stripe.save' | t }}</button>
     </form>
   </div>
+  </br>
+
+  <div class="mb-2 w-full md:w-full">
+    <h1 class="text-2xl">Chat</h1>
+  </div>
+
+  <div class="w-full p-6 mt-8 mb-2 bg-gray-200 rounded lg:w-4/6 md:w-3/4">
+    <form action="/marketplace_setup/constants/set" method="post">
+      {% include 'authenticity_token' %}
+      <input type="hidden" name="name" value="chat">
+      <input type="radio" class="mt-3" name="value" value="true" id="chatOn"{% if chat_enabled %}checked{% endif %}>
+      <label class="block mt-4" for="chatOn">enable</label>
+      <input type="radio" class="mt-3" name="value" value="false" id="chatOff"{% unless chat_enabled %}checked{% endunless %}>
+      <label class="block mt-4" for="chatOff">disable</label>
+      <input type="submit" value="submit" class="mt-3 button">
+    </form>
+  </div>
+
+  <div class="mb-2 w-full md:w-full">
+    <h1 class="text-2xl">{{ 'app.slack.setup.title' | t }}</h1>
+  </div>
+
+  <div class="w-full p-6 mt-8 mb-2 bg-gray-200 rounded lg:w-4/6 md:w-3/4">
+    <form action="/marketplace_setup/constants/set" method="post">
+      {% include 'authenticity_token' %}
+      <label class="block mt-4">
+      <input type="hidden" name="name" value="slack"></input>
+      <input type="text" name="value" value=""></input>
+      <input type="submit" class="ml-3 button button" value="submit" class="mt-3 button"></br>
+      <label class="block mt-4"><b>{{ 'app.slack.setup.webhook' | t }}</b></br> {% if context.constants.SLACK_WEBHOOK_URL == blank %}{{ 'app.slack.setup.none' | t }}{% else %}{{ context.constants.SLACK_WEBHOOK_URL }}{% endif %}</label>
+      </label>
```

Comment:
This html is very weird.

It opens label... then puts inputs and labels inside, then closes label.

IMO labels should no be nested, probably w3 standard thinks that as well.